test/tests/Localisation.cpp

```diff
@@ -7,9 +7,8 @@
  * OpenRCT2 is licensed under the GNU General Public License version 3.
  *****************************************************************************/
 
-#include "openrct2/localisation/Localisation.h"
-
 #include "helpers/StringHelpers.hpp"
+#include "openrct2/localisation/Language.h"
```

Comment:
I can't figure out why this file is not found, even though Localisation.h appears to have been no problem. There doesn't appear to be a whitelist for headers in CMakeLists.txt, so... why?

Reply:
It seems tests are actually running fine, but it's just clang-tidy that's complaining.